File: IT/1/Oggetti/Java/Person2.java
/* file Person2.java */

class Person {
	/* definizione dei campi "name" e "surname" */
	public String name;
	public String surname;

	/* definizione del costruttore della classe, che inizializza "name" e "surname" */
	public Person(String name, String surname) {
		this.name = name;
		this.surname = surname;
	}
}

/* la classe Env contiene il main() da cui inizia l'esecuzione del programma */	

class Env {
	public static void main(String [] args) {
		/* crea un oggetto di tipo "persona" */
		Person p = new Person("John", "Doe");
		System.out.println("Il nostro uomo si chiama " + p.name + " " + p.surname + ".");
	}
}

File: IT/1/Oggetti/Python/Person2.py
# file Person2.py

from __future__ import print_function

class Person:
	# dichiarazione del metodo __init__(), che inizializza i campi "name" e "surname"
	def __init__(self, name, surname):
		self.name = name
		self.surname = surname

# crea un oggetto di tipo persona
p = Person("John", "Doe")
print("Il nostro uomo si chiama {} {}.".format(p.name, p.surname))

